awx/main/tasks.py

```diff
@@ -1182,9 +1186,12 @@ def run(self, pk, **kwargs):
                 manager_instance = isolated_manager.IsolatedManager(env, **_kw)
                 status, rc = manager_instance.run(self.instance,
                                                   private_data_dir,
-                                                  playbook,
+                                                  params.get('playbook'),
+                                                  params.get('module'),
+                                                  module_args,
                                                   event_data_key=self.event_data_key,
                                                   ident=str(self.instance.pk))
+                self.event_ct = len(manager_instance.handled_events)
```

Comment:
@chrismeyersfsu we've also got to get the count of handled events from the iso code so that `event_processing_finished` goes to `True`.  This was causing a lot of isolated test failures.